src/utilities/gstat/dba.epp

```diff
@@ -612,9 +612,9 @@ int gstat(Firebird::UtilSvc* uSvc)
 
 	dba_fil* current = db_open(fileName.c_str(), fileName.length());
 
-	SCHAR temp[RAW_HEADER_SIZE];
-	tddba->page_size = sizeof(temp);
-	tddba->global_buffer = (pag*) temp;
+	SCHAR temp[RAW_HEADER_SIZE + DIRECT_IO_BLOCK_SIZE];
```

Comment:
You can use `alignas(DIRECT_IO_BLOCK_SIZE)` here.

Reply:
Done